docs/tests-ui.rst

```diff
@@ -1,51 +1,89 @@
 Client-side Testing with Intern
 ===============================
 
-The client-side testing tool for the MDN front-end is `Intern <https://theintern.github.io/>`_, a Selenium WebDriver API which allows developers to write automated testing via JavaScript. Intern is an open source project created and maintained by `SitePen <http://sitepen.com>`_.
+We use `Intern <https://theintern.github.io/>`_ for client-side testing. It uses Selenium WebDriver API which lets us write automated testing via JavaScript. Intern is an open source project created and maintained by `SitePen <http://sitepen.com>`_.
 
 Installing Dependencies
 -----------------------
 
-1. Install `JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_
-
-2. Download the most current release of Selenium `WebDriver <http://selenium-release.storage.googleapis.com/index.html>`_. Download the current standalone version which is a `.jar` file.
+1. Go to the ``tests/ui/`` directory::
 
-3. From the `tests/ui/` directory, use NPM or another package manager to install Intern::
+    cd tests/ui
+    
+2. Use ``npm`` to install Intern::
 
     npm install intern@^3.0
 
-Do *not* install Intern globally -- path issues may occur.
-
-Firefox appears to work out of the box, but `Chrome <https://sites.google.com/a/chromium.org/chromedriver/>`_ and `Safari <https://code.google.com/p/selenium/wiki/SafariDriver>`_ drivers must be downloaded and installed separately.
+.. warning:: Do *not* install Intern globally -- path issues may occur.
```

Comment:
Just a formatting nitpick. The word *Warning* is showing up as its own paragraph for some reason.